README.md

```diff
@@ -21,7 +22,7 @@ That process is documented in the [solrizer README](https://github.com/projecthy
 
 ## OM in the Wild
 
-We have a page on the Hydra wiki with a list of OM Terminologies in active use: 
+We have a page on the Hydra wiki with a list of OM Terminologies in active use:
```

Comment:
I only spotted this because of the whitespace change -- the text should be updated to Samvera. However, I cannot find the confluence page that was supposed to be linked here. The content has apparently been moved or superseded. This is probably an issue for @joncameron.

Reply:
Thanks @botimer, I saw that @jrgriffiniii nixed that bit in the last PR. I think that page is lost to time—and probably super out date if it still exists somewhere out there—so having that bit removed seems best.